bundles/admin/admin-layereditor/view/AdminLayerForm/GeneralTabPane/DataProvider.jsx

```diff
@@ -9,7 +9,7 @@ export const DataProvider = ({ layer, dataProviders, controller }) => (
         <Message messageKey='dataProvider'/>
         <StyledComponent>
             <Select
-                value={layer.organizationName}
+                value={layer.groupId}
```

Comment:
Should this be dataProviderId instead so it's not confused to layer group id?

Reply:
Yes, I'll refactor it. I'll also change mapLayerGroups to just groups on layer data.